updatecontrollerpackagekit.cpp

```diff
@@ -492,6 +492,7 @@ void UpdateControllerPackageKit::readDistro()
     knownDistros.insert("21.10", "impish");
     knownDistros.insert("22.04", "jammy");
     knownDistros.insert("22.10", "kinetic");
+    knownDistros.insert("23.04", "lunar");
     // Debian
     knownDistros.insert("9", "stretch");
```

Comment:
I think we can remove the stretch...

Reply:
There are actually more that are EOL... Half of the ubuntu ones could go away too I suppose... Was thinking about it too but decided to not do that in this PR for no particular reason.